cluster/senza-definition.yaml

```diff
@@ -92,6 +94,25 @@ SenzaComponents:
       Tags:
         - Key: "KubernetesCluster"
           Value: kubernetes
+  - WorkerLoadBalancer:
+      Type: Senza::WeightedDnsElasticLoadBalancer
+      NameSuffix: worker
+      MainDomain: ignore-main-{{SenzaInfo.StackVersion}}.teapot.zalan.do
+      VersionDomain: ignore-version-{{SenzaInfo.StackVersion}}.teapot.zalan.do
+      HTTPPort: 4194
+      HealthCheckPath: /healthz
+      HealthCheckPort: 4194
+      Listeners:
+        - PolicyNames: []
+          Protocol: HTTP
+          InstanceProtocol: HTTP
+          InstancePort: 4194
+          LoadBalancerPort: 4194
+      Tags:
+        - Key: "Teapot"
+          Value: instance health check
```

Comment:
Could also mean "this is owned by the cluster". Ideally the `kubernetesCluster` tag value should be different for each cluster in an account so it's easy to identify which resource is owned by which cluster.

I know we currently don't do this because of issues with multi AZ iirc, but hopefully in the future :)

We don't have to discuss it now. If you don't want to add it for the reasons you put then I'm fine with it :)